o2a/o2a_libs/el_parser.py

```diff
@@ -116,9 +116,9 @@
 
     PARAM: /(?!true|false|null)([a-zA-Z_]+)/
 
-    JAVA: /(?!true|false|null)([a-zA-Z_]+)/
+    JAVA: /(?!true|false|null)([a-zA-Z_]+[0-9]*)/
 
-    FIRST_JAVA: /(?!true|false|null)([a-zA-Z_]+)/
+    FIRST_JAVA: /(?!true|false|null)([a-zA-Z_]+[0-9]*)/
```

Comment:
How about this: `[a-zA-Z_\$][a-zA-Z0-9_\$]*`  - I think this is much better regex reflecting the actual java language specification: https://stackoverflow.com/a/4964751/516701